docker/app/000-default.conf

```diff
@@ -0,0 +1,35 @@
+ServerName 127.0.0.1
+
+<VirtualHost *:80>
+    DocumentRoot /var/www/web
+
+    <Directory /var/www/web>
+        Require all granted
+        DirectoryIndex app.php
```

Comment:
I prefer app_dev.php if you can change this, just to be easier for the newcomers.